### ISSUE TITLE
Unable to create new personalized questions

### ISSUE THREAD

**Issue by jonymusky** [metabase/metabase#12323]
**Describe the bug**
After upgrading to the last version of metabase. 
Saving question -> not working.
Creating a new question -> is not loading with the log that I sent.

![image](https://user-images.githubusercontent.com/414404/79268248-d13b9000-7e70-11ea-9dd9-ac7ea78c6df7.png)
![image](https://user-images.githubusercontent.com/414404/79268351-08aa3c80-7e71-11ea-9e26-e1b402d18564.png)

**Logs**
```
2020-04-14T16:43:30-03:00 ERROR metabase.middleware.log GET /api/database 500 15.6 ms (5 DB calls)
{:message "ERROR: syntax error at or near \"NULL\"\n Position: 303",
:type org.postgresql.util.PSQLException,
:stacktrace
["org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2497)"
"org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2233)"
"org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)"
"org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:446)"
"org.postgresql.jdbc.PgStatement.execute(PgStatement.java:370)"
"org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:149)"
"org.postgresql.jdbc.PgPreparedStatement.executeQuery(PgPreparedStatement.java:108)"
"com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeQuery(NewProxyPreparedStatement.java:431)"
"clojure.java.jdbc$execute_query_with_params.invokeStatic(jdbc.clj:1072)"
"clojure.java.jdbc$execute_query_with_params.invoke(jdbc.clj:1066)"
"clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1095)"
"clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:1075)"
"clojure.java.jdbc$reducible_query$reify__18756.reduce(jdbc.clj:1339)"
"clojure.core$transduce.invokeStatic(core.clj:6883)"
"clojure.core.Eduction.reduce(core.clj:7682)"
"clojure.core$transduce.invokeStatic(core.clj:6883)"
"clojure.core$transduce.invoke(core.clj:6870)"
"--> api.database$source_query_cards.invokeStatic(database.clj:126)"
"api.database$source_query_cards.doInvoke(database.clj:123)"
"api.database$source_query_cards_exist_QMARK_.invokeStatic(database.clj:143)"
"api.database$source_query_cards_exist_QMARK_.invoke(database.clj:140)"
"api.database$add_saved_questions_virtual_database.invokeStatic(database.clj:166)"
"api.database$add_saved_questions_virtual_database.doInvoke(database.clj:162)"
"api.database$dbs_list.invokeStatic(database.clj:172)"
"api.database$dbs_list.doInvoke(database.clj:168)"
"api.database$fn__54399.invokeStatic(database.clj:207)"
"api.database$fn__54399.invoke(database.clj:174)"
"middleware.auth$enforce_authentication$fn__66831.invoke(auth.clj:14)"
"routes$fn__68355$fn__68356.doInvoke(routes.clj:56)"
"middleware.exceptions$catch_uncaught_exceptions$fn__66930.invoke(exceptions.clj:97)"
"middleware.exceptions$catch_api_exceptions$fn__66927.invoke(exceptions.clj:85)"
"middleware.log$log_api_call$fn__68733$fn__68734.invoke(log.clj:197)"
"middleware.log$log_api_call$fn__68733.invoke(log.clj:191)"
"middleware.security$add_security_headers$fn__66893.invoke(security.clj:121)"
"middleware.json$wrap_json_body$fn__68436.invoke(json.clj:66)"
"middleware.json$wrap_streamed_json_response$fn__68454.invoke(json.clj:100)"
"middleware.misc$maybe_set_site_url$fn__68763.invoke(misc.clj:59)"
"middleware.session$bind_current_user$fn__63420$fn__63421.invoke(session.clj:209)"
"middleware.session$do_with_current_user.invokeStatic(session.clj:184)"
"middleware.session$do_with_current_user.invoke(session.clj:177)"
"middleware.session$bind_current_user$fn__63420.invoke(session.clj:208)"
"middleware.session$wrap_current_user_id$fn__63405.invoke(session.clj:161)"
"middleware.session$wrap_session_id$fn__63349.invoke(session.clj:125)"
"middleware.auth$wrap_api_key$fn__66839.invoke(auth.clj:27)"
"middleware.misc$bind_user_locale$fn__68768.invoke(misc.clj:78)"
"middleware.misc$add_content_type$fn__68749.invoke(misc.clj:29)"
"middleware.misc$disable_streaming_buffering$fn__68777.invoke(misc.clj:94)"
"server$async_proxy_handler$fn__68511.invoke(server.clj:70)"
"server.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle(Unknown Source)"],
:sql-exception-chain ["PSQLException:" "Message: ERROR: syntax error at or near \"NULL\"" "Position: 303" "SQLState: 42601" "Error Code: 0"]}
```

**To Reproduce**
1. Go to '

**Expected behavior**

**Screenshots**

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

> {
>   "browser-info": {
>     "language": "es-419",
>     "platform": "MacIntel",
>     "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36",
>     "vendor": "Google Inc."
>   },
>   "system-info": {
>     "file.encoding": "UTF-8",
>     "java.runtime.name": "OpenJDK Runtime Environment",
>     "java.runtime.version": "1.8.0_242-heroku-b08",
>     "java.vendor": "Oracle Corporation",
>     "java.vendor.url": "http://java.oracle.com/",
>     "java.version": "1.8.0_242-heroku",
>     "java.vm.name": "OpenJDK 64-Bit Server VM",
>     "java.vm.version": "25.242-b08",
>     "os.name": "Linux",
>     "os.version": "4.4.0-1062-aws",
>     "user.language": "en",
>     "user.timezone": "Etc/UTC"
>   },
>   "metabase-info": {
>     "databases": [
>       "googleanalytics",
>       "mongo"
>     ],
>     "hosting-env": "heroku",
>     "application-database": "postgres",
>     "application-database-details": {
>       "database": {
>         "name": "PostgreSQL",
>         "version": "10.11 (Ubuntu 10.11-1.pgdg16.04+1)"
>       },
>       "jdbc-driver": {
>         "name": "PostgreSQL JDBC Driver",
>         "version": "42.2.8"
>       }
>     },
>     "run-mode": "prod",
>     "version": {
>       "tag": "v0.35.2",
>       "date": "2020-04-10",
>       "branch": "release-0.35.x",
>       "hash": "f3b2857"
>     },
>     "settings": {
>       "report-timezone": "US/Eastern"
>     }
>   }
> }

- Your operating system: Osx
- Your databases: MongoDB, Google Analytics
- Metabase version: 0.35.2
- Metabase hosting environment: Heroku
- Metabase internal database: Postgres

**Severity**
This is very important because we can't create new things.

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by flamber** [comment]
Hi @jonymusky 
Possible workaround: When you see the spinner, then click "Browse Data" once, and then it should load the databases again, when clicking back to "Ask a question"

I'm not sure if this is just more visible on Heroku, since how it handles inactivity on the instance.
It was reported in #12137, but I thought it was specific to 0.35.0-RC1, and wasn't reported again.

I've noticed something similar on my dev machine (H2 appdb), but I thought it was just my setup:
1. Start `lein run`
2. Open a new tab and goto `http://localhost:3000/`
3. Click "Ask a question" - shows spinner forever, while I'm seeing a different error (actually two, one of sqlite and sqlserver, but I doubt it's related to those databases, it's just the databases I have configured in the instance besides the core ones, H2/Postgres/MySQL):
```
04-14 22:24:53 ERROR middleware.log :: GET /api/database 500 134.2 ms (5 DB calls) 
{:classloader #object[clojure.lang.DynamicClassLoader 0x29aa4bc9 "clojure.lang.DynamicClassLoader@29aa4bc9"],
 :classpath-urls (),
 :message "Could not locate metabase/driver/sqlite__init.class, metabase/driver/sqlite.clj or metabase/driver/sqlite.cljc on classpath.",
 :type clojure.lang.ExceptionInfo,
 :stacktrace
 ["--> plugins.classloader$require.invokeStatic(classloader.clj:128)"
  "plugins.classloader$require.doInvoke(classloader.clj:113)"
  "plugins.init_steps$eval79794$fn__79796.invoke(init_steps.clj:22)"
  "plugins.init_steps$do_init_steps_BANG_.invokeStatic(init_steps.clj:32)"
  "plugins.init_steps$do_init_steps_BANG_.invoke(init_steps.clj:27)"
  "plugins.lazy_loaded_driver$make_initialize_BANG_$fn__79895.invoke(lazy_loaded_driver.clj:58)"
  "driver.impl$initialize_if_needed_BANG_.invokeStatic(impl.clj:179)"
  "driver.impl$initialize_if_needed_BANG_.invoke(impl.clj:160)"
  "driver$the_initialized_driver.invokeStatic(driver.clj:147)"
  "driver$the_initialized_driver.invoke(driver.clj:143)"
  "driver$dispatch_on_initialized_driver.invokeStatic(driver.clj:155)"
  "driver$dispatch_on_initialized_driver.doInvoke(driver.clj:150)"
  "driver$eval27550$fn__27551.invoke(driver.clj:413)"
  "api.database$ids_of_dbs_that_support_source_queries$fn__63642.invoke(database.clj:120)"
  "api.database$ids_of_dbs_that_support_source_queries.invokeStatic(database.clj:119)"
  "api.database$ids_of_dbs_that_support_source_queries.invoke(database.clj:118)"
  "api.database$source_query_cards.invokeStatic(database.clj:134)"
  "api.database$source_query_cards.doInvoke(database.clj:123)"
  "api.database$source_query_cards_exist_QMARK_.invokeStatic(database.clj:143)"
  "api.database$source_query_cards_exist_QMARK_.invoke(database.clj:140)"
  "api.database$add_saved_questions_virtual_database.invokeStatic(database.clj:166)"
  "api.database$add_saved_questions_virtual_database.doInvoke(database.clj:162)"
  "api.database$dbs_list.invokeStatic(database.clj:172)"
  "api.database$dbs_list.doInvoke(database.clj:168)"
  "api.database$fn__63689.invokeStatic(database.clj:207)"
  "api.database$fn__63689.invoke(database.clj:174)"
  "middleware.auth$enforce_authentication$fn__78218.invoke(auth.clj:14)"
  "routes$fn__78476$fn__78477.doInvoke(routes.clj:56)"
  "middleware.exceptions$catch_uncaught_exceptions$fn__78339.invoke(exceptions.clj:97)"
  "middleware.exceptions$catch_api_exceptions$fn__78336.invoke(exceptions.clj:85)"
  "middleware.log$log_api_call$fn__78902$fn__78903.invoke(log.clj:197)"
  "middleware.log$log_api_call$fn__78902.invoke(log.clj:191)"
  "middleware.security$add_security_headers$fn__78294.invoke(security.clj:121)"
  "middleware.json$wrap_json_body$fn__78583.invoke(json.clj:64)"
  "middleware.json$wrap_streamed_json_response$fn__78601.invoke(json.clj:100)"
  "middleware.misc$maybe_set_site_url$fn__78940.invoke(misc.clj:59)"
  "middleware.session$bind_current_user$fn__74145$fn__74146.invoke(session.clj:209)"
  "middleware.session$do_with_current_user.invokeStatic(session.clj:184)"
  "middleware.session$do_with_current_user.invoke(session.clj:177)"
  "middleware.session$bind_current_user$fn__74145.invoke(session.clj:208)"
  "middleware.session$wrap_current_user_id$fn__74122.invoke(session.clj:161)"
  "middleware.session$wrap_session_id$fn__74066.invoke(session.clj:125)"
  "middleware.auth$wrap_api_key$fn__78226.invoke(auth.clj:27)"
  "middleware.misc$bind_user_locale$fn__78945.invoke(misc.clj:78)"
  "middleware.misc$add_content_type$fn__78926.invoke(misc.clj:29)"
  "middleware.misc$disable_streaming_buffering$fn__78954.invoke(misc.clj:94)"
  "server$async_proxy_handler$fn__78678.invoke(server.clj:70)"
  "server.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle(Unknown Source)"]}
```
4. Clicking "Browse Data" and then back to "Ask a question" will load the databases correctly (doing a browser refresh on "Ask a question" also works). This works until next time I restart the instance.

**Comment by jonymusky** [comment]
Hi @flamber its a paid dyno with 2.5gb of RAM we are not using a dyno that goes to sleep on inactivity. We also pay for our Postgres instance.

Restarting the dyno didn't work neither refreshing the page or going back.

**Comment by flamber** [comment]
@jonymusky I don't know how much logging you have access to on the Postgres instance, but it would be really great if you could figure out if there's a more complete error message there, or the actually query that is failing.

**Comment by flamber** [comment]
_Originally posted by @mhjb in https://github.com/metabase/metabase/issues/12329#issuecomment-614285724_
> Here are the loading spinner logs:
> ![image](https://user-images.githubusercontent.com/1380991/79390062-21c30280-7fc3-11ea-8bcf-0863f98a6416.png)
> 
> ```
> 04-15 21:18:49 ERROR middleware.log :: GET /api/database 500 7.2 ms (7 DB calls)
> {:message
>  "Syntax error in SQL statement \"SELECT \"\"REPORT_CARD\"\".\"\"NAME\"\", \"\"REPORT_CARD\"\".\"\"DESCRIPTION\"\", \"\"REPORT_CARD\"\".\"\"DATABASE_ID\"\", \"\"REPORT_CARD\"\".\"\"DATASET_QUERY\"\", \"\"REPORT_CARD\"\".\"\"ID\"\", \"\"REPORT_CARD\"\".\"\"COLLECTION_ID\"\", \"\"REPORT_CARD\"\".\"\"RESULT_METADATA\"\" FROM \"\"REPORT_CARD\"\" WHERE (\"\"RESULT_METADATA\"\" IS NOT NULL AND \"\"ARCHIVED\"\" = FALSE AND (\"\"DATABASE_ID\"\" IN NULL[*]) AND TRUE) ORDER BY LOWER(\"\"NAME\"\") ASC \"; expected \"(\"; SQL statement:\nSELECT \"REPORT_CARD\".\"NAME\", \"REPORT_CARD\".\"DESCRIPTION\", \"REPORT_CARD\".\"DATABASE_ID\", \"REPORT_CARD\".\"DATASET_QUERY\", \"REPORT_CARD\".\"ID\", \"REPORT_CARD\".\"COLLECTION_ID\", \"REPORT_CARD\".\"RESULT_METADATA\" FROM \"REPORT_CARD\" WHERE (\"RESULT_METADATA\" IS NOT NULL AND \"ARCHIVED\" = FALSE AND (\"DATABASE_ID\" in NULL) AND TRUE) ORDER BY lower(\"NAME\") ASC [42001-197]",
>  :type org.h2.jdbc.JdbcSQLException,
>  :stacktrace
>  ["org.h2.message.DbException.getJdbcSQLException(DbException.java:357)"
>   "org.h2.message.DbException.getSyntaxError(DbException.java:217)"
>   "org.h2.command.Parser.getSyntaxError(Parser.java:555)"
>   "org.h2.command.Parser.read(Parser.java:3518)"
>   "org.h2.command.Parser.readCondition(Parser.java:2433)"
>   "org.h2.command.Parser.readAnd(Parser.java:2342)"
>   "org.h2.command.Parser.readExpression(Parser.java:2334)"
>   "org.h2.command.Parser.readTerm(Parser.java:3252)"
>   "org.h2.command.Parser.readFactor(Parser.java:2587)"
>   "org.h2.command.Parser.readSum(Parser.java:2574)"
>   "org.h2.command.Parser.readConcat(Parser.java:2544)"
>   "org.h2.command.Parser.readCondition(Parser.java:2370)"
>   "org.h2.command.Parser.readAnd(Parser.java:2344)"
>   "org.h2.command.Parser.readExpression(Parser.java:2334)"
>   "org.h2.command.Parser.readTerm(Parser.java:3252)"
>   "org.h2.command.Parser.readFactor(Parser.java:2587)"
>   "org.h2.command.Parser.readSum(Parser.java:2574)"
>   "org.h2.command.Parser.readConcat(Parser.java:2544)"
>   "org.h2.command.Parser.readCondition(Parser.java:2370)"
>   "org.h2.command.Parser.readAnd(Parser.java:2342)"
>   "org.h2.command.Parser.readExpression(Parser.java:2334)"
>   "org.h2.command.Parser.parseSelectSimple(Parser.java:2291)"
>   "org.h2.command.Parser.parseSelectSub(Parser.java:2133)"
>   "org.h2.command.Parser.parseSelectUnion(Parser.java:1946)"
>   "org.h2.command.Parser.parseSelect(Parser.java:1919)"
>   "org.h2.command.Parser.parsePrepared(Parser.java:463)"
>   "org.h2.command.Parser.parse(Parser.java:335)"
>   "org.h2.command.Parser.parse(Parser.java:311)"
>   "org.h2.command.Parser.prepareCommand(Parser.java:278)"
>   "org.h2.engine.Session.prepareLocal(Session.java:611)"
>   "org.h2.engine.Session.prepareCommand(Session.java:549)"
>   "org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)"
>   "org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)"
>   "org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:304)"
>   "com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:387)"
>   "clojure.java.jdbc$prepare_statement.invokeStatic(jdbc.clj:672)"
>   "clojure.java.jdbc$prepare_statement.invoke(jdbc.clj:619)"
>   "clojure.java.jdbc$db_query_with_resultset_STAR_.invokeStatic(jdbc.clj:1094)"
>   "clojure.java.jdbc$db_query_with_resultset_STAR_.invoke(jdbc.clj:1075)"
>   "clojure.java.jdbc$reducible_query$reify__18756.reduce(jdbc.clj:1339)"
>   "clojure.core$transduce.invokeStatic(core.clj:6883)"
>   "clojure.core.Eduction.reduce(core.clj:7682)"
>   "clojure.core$transduce.invokeStatic(core.clj:6883)"
>   "clojure.core$transduce.invoke(core.clj:6870)"
>   "--> api.database$source_query_cards.invokeStatic(database.clj:126)"
>   "api.database$source_query_cards.doInvoke(database.clj:123)"
>   "api.database$source_query_cards_exist_QMARK_.invokeStatic(database.clj:143)"
>   "api.database$source_query_cards_exist_QMARK_.invoke(database.clj:140)"
>   "api.database$add_saved_questions_virtual_database.invokeStatic(database.clj:166)"
>   "api.database$add_saved_questions_virtual_database.doInvoke(database.clj:162)"
>   "api.database$dbs_list.invokeStatic(database.clj:172)"
>   "api.database$dbs_list.doInvoke(database.clj:168)"
>   "api.database$fn__54399.invokeStatic(database.clj:207)"
>   "api.database$fn__54399.invoke(database.clj:174)"
>   "middleware.auth$enforce_authentication$fn__66831.invoke(auth.clj:14)"
>   "routes$fn__68355$fn__68356.doInvoke(routes.clj:56)"
>   "middleware.exceptions$catch_uncaught_exceptions$fn__66930.invoke(exceptions.clj:97)"
>   "middleware.exceptions$catch_api_exceptions$fn__66927.invoke(exceptions.clj:85)"
>   "middleware.log$log_api_call$fn__68733$fn__68734.invoke(log.clj:197)"
>   "middleware.log$log_api_call$fn__68733.invoke(log.clj:191)"
>   "middleware.security$add_security_headers$fn__66893.invoke(security.clj:121)"
>   "middleware.json$wrap_json_body$fn__68436.invoke(json.clj:64)"
>   "middleware.json$wrap_streamed_json_response$fn__68454.invoke(json.clj:100)"
>   "middleware.misc$maybe_set_site_url$fn__68763.invoke(misc.clj:59)"
>   "middleware.session$bind_current_user$fn__63420$fn__63421.invoke(session.clj:209)"
>   "middleware.session$do_with_current_user.invokeStatic(session.clj:184)"
>   "middleware.session$do_with_current_user.invoke(session.clj:177)"
>   "middleware.session$bind_current_user$fn__63420.invoke(session.clj:208)"
>   "middleware.session$wrap_current_user_id$fn__63405.invoke(session.clj:161)"
>   "middleware.session$wrap_session_id$fn__63349.invoke(session.clj:125)"
>   "middleware.auth$wrap_api_key$fn__66839.invoke(auth.clj:27)"
>   "middleware.misc$bind_user_locale$fn__68768.invoke(misc.clj:78)"
>   "middleware.misc$add_content_type$fn__68749.invoke(misc.clj:29)"
>   "middleware.misc$disable_streaming_buffering$fn__68777.invoke(misc.clj:94)"
>   "server$async_proxy_handler$fn__68511.invoke(server.clj:70)"
>   "server.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a.handle(Unknown Source)"],
>  :sql-exception-chain
>  ["JdbcSQLException:"
>   "Message: Syntax error in SQL statement \"SELECT \"\"REPORT_CARD\"\".\"\"NAME\"\", \"\"REPORT_CARD\"\".\"\"DESCRIPTION\"\", \"\"REPORT_CARD\"\".\"\"DATABASE_ID\"\", \"\"REPORT_CARD\"\".\"\"DATASET_QUERY\"\", \"\"REPORT_CARD\"\".\"\"ID\"\", \"\"REPORT_CARD\"\".\"\"COLLECTION_ID\"\", \"\"REPORT_CARD\"\".\"\"RESULT_METADATA\"\" FROM \"\"REPORT_CARD\"\" WHERE (\"\"RESULT_METADATA\"\" IS NOT NULL AND \"\"ARCHIVED\"\" = FALSE AND (\"\"DATABASE_ID\"\" IN NULL[*]) AND TRUE) ORDER BY LOWER(\"\"NAME\"\") ASC \"; expected \"(\"; SQL statement:"
>   "SELECT \"REPORT_CARD\".\"NAME\", \"REPORT_CARD\".\"DESCRIPTION\", \"REPORT_CARD\".\"DATABASE_ID\", \"REPORT_CARD\".\"DATASET_QUERY\", \"REPORT_CARD\".\"ID\", \"REPORT_CARD\".\"COLLECTION_ID\", \"REPORT_CARD\".\"RESULT_METADATA\" FROM \"REPORT_CARD\" WHERE (\"RESULT_METADATA\" IS NOT NULL AND \"ARCHIVED\" = FALSE AND (\"DATABASE_ID\" in NULL) AND TRUE) ORDER BY lower(\"NAME\") ASC [42001-197]"
>   "SQLState: 42001"
>   "Error Code: 42001"]}
> ```

**Comment by flamber** [comment]
I'm still not able to reproduce this on a regular release, but it's been reported that the problem on occurs on non-H2 backends: https://discourse.metabase.com/t/cannot-display-schema-list-0-35-x/9921

**Comment by flamber** [comment]
:tada: I was able to reproduce it, thanks to [oke](https://discourse.metabase.com/t/cannot-display-schema-list-0-35-x/9921/9?u=flamber) in the forum.

This only breaks, when Admin > Settings > General > "Enable Nested Queries" is disabled.
**Workaround** Enable the setting until this issue is fixed.

I've now seen two different types of breakage, but I haven't seen the database query errors as the other comments:
1. Internal instance (almost) 0.35.3 with Postgres backend shows loading spinner and there's a browser console error: `TypeError: e is null`:
![image](https://user-images.githubusercontent.com/1447303/80795875-0ef12600-8b9e-11ea-9b92-0e83d601a679.png)
2. Local instance 0.35.3 with MariaDB backend shows nothing, but there's a browser console error: `TypeError: e is null`:
![image](https://user-images.githubusercontent.com/1447303/80796130-ba9a7600-8b9e-11ea-8a84-0311e851b06b.png)
3. Another local 0.35.3 on H2 does the same as 2)

_Removing the Heroku label, but leaving Backend, since I'm not sure if that is still related_

**Comment by flamber** [comment]
@sbelak I think this has fixed it for those seeing database errors, but it's not fixed if you turn off the setting Admin > Settings > General > "Enable Nested Queries".
On commit a8611dfcd7099274fc3fea57919e31bf20ee0758 I'm still seeing the empty selector with a bunch of JS errors - I guess this is now a FE issue?
[Chromium-1589397849868.log](https://github.com/metabase/metabase/files/4624244/localhost-1589397849868.log)

**Comment by camsaul** [comment]
Fixed by #12527 and #12530
